playbooks/byo/openshift-prometheus.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
openshift-ansible is in the middle of refactoring the way they do these top-level byo playbooks. look to https://github.com/openshift/openshift-ansible/blob/master/playbooks/byo/openshift-cluster/config.yml for an example of how they are doing things now.

i'm not 100% familiar with all the details, but someone on the openshift-ansible team should be able to provide more info.